app/models/prison.rb

```diff
@@ -38,6 +38,10 @@ def allocations_for(staff_id)
     }
   end
 
+  def self.all
```

Comment:
as previously discussed, 'all' isn't a good thing to query on' as it included prisons which are not live

Reply:
I'll remove the `Prison.all` method as it's now unused (as no longer used by the rake task).